code/modules/mob/living/silicon/robot/drone/drone_abilities.dm

```diff
@@ -74,7 +74,11 @@
 		to_chat(src, "<span class='warning'>Error 404: Custom chassis not found. Revoking customization option.</span>")
 	else
 		icon = 'icons/mob/custom_synthetic/custom-synthetic.dmi'
-		icon_state = "[ckey]-drone"
+		if(emagged)			//prevents someone from changing their sprite to their default custom sprite when emagged
+//			icon_state = "[ckey]-drone-emagged"		//in case custom emagged drones are allowed, comment out the line bellow and uncomment this line
```

Comment:
I'd either just make customization not work if emagged or require an emagged sprite be supplied. Consistency would be better than commented out code being added "just in case"